docker/docker-compose.yml

```diff
@@ -21,6 +21,7 @@ services:
     environment: 
       - POSTGRES_USER=postgres
       - POSTGRES_PASSWORD=0000
+      - PGDATA=/tmp
```

Comment:
Necessary to avoid to PostgreSQL fail on build. See https://github.com/laradock/laradock/issues/919